packages/services/document-server/src/main/java/software/uncharted/terarium/documentserver/proxies/xdd/DocumentProxy.java

```diff
@@ -0,0 +1,50 @@
+package software.uncharted.terarium.documentserver.proxies.xdd;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@RegisterRestClient(configKey = "xdd-document-service")
```

Comment:
Is this where you make reference to the xDD API URL in the `resources/application.properties`?
Then _Microprofile_ automaticaly does the REST call with the `@GET` annotation?